docs/Custom-iOS.md

```diff
@@ -132,6 +132,16 @@ Once these are exposed, you can reference them in your custom web view class.
 }
 ```
 
+### Setting Client Certificate Authentication Credential
+
+If you open webpages that needs a Client Certificate for Authentication, you can create a credential and pass it to the webview:
+
+```
+[RNCWKWebView setClientAuthenticationCredential:credential];
```

Comment:
@mjhu  Can you explain where you would put this code, normally? Would it be in the `AppDelegate.m` file?

Reply:
It doesn't have to be in AppDelegate.m. For our use case, we have this call inside our auth initiation stack that runs as part of app start. The only requirement is that call is made before using any webviews. I'll update the comment with this.